core/primitives/src/version.rs

```diff
@@ -259,6 +261,8 @@ impl ProtocolFeature {
             ProtocolFeature::FixContractLoadingCost => 129,
             #[cfg(feature = "protocol_feature_account_id_in_function_call_permission")]
             ProtocolFeature::AccountIdInFunctionCallPermission => 130,
+            #[cfg(feature = "protocol_feature_delegate_action")]
+            ProtocolFeature::DelegateAction => 131,
```

Comment:
Quite a bit has changed on master, you will have to pick a bigger number when merging, 133 seems to be the newest. Also, you need to update `pub const PROTOCOL_VERSION:` to the new largest number for nightly.